weed/storage/volume_vacuum.go

```diff
@@ -55,7 +71,85 @@ func (v *Volume) commitCompact() error {
 	return nil
 }
 
-func makeupDiff(newDatFile, newIdxFile, oldDatFile, oldIdxFile string) (err error) {
+func (v *Volume) makeupDiff(newDatFileName, newIdxFileName, oldDatFileName, oldIdxFileName string) (err error) {
```

Comment:
Is there some good ways to do it? I 'm thinking about creating 4 test volume/index files in advance(new creating, updating and deleting appended data should be in them), then write a test case  with these volume files as input